addons/project/wizard/project_share_wizard.py

```diff
@@ -11,7 +11,14 @@ class ProjectShareWizard(models.TransientModel):
 
     @api.model
     def default_get(self, fields):
-        result = super().default_get(fields)
+        # We pass active_id and active_model in context because super method gets value in context.
+        ctx = dict(self._context)
+        default_res_model = self._context.get('default_res_model')
+        default_res_id = self._context.get('default_res_id')
+        default_access_mode = self._context.get('default_access_mode')
+        if default_res_model and default_res_id:
+            ctx.update({'active_model': default_res_model, 'active_id': default_res_id, 'default_access_mode': default_access_mode, })
```

Comment:
Why not giving `active_model` and `active_id` in the context of the button? 🤔

Reply:
Hello @xavierbol 
The active_id and active_model are being overridden by the parent method (Link: https://github.com/odoo/odoo/blob/16.0/addons/portal/wizard/portal_share.py#LL11C5-L11C35), even when I pass them through the context of the button.
So updating the context here instead of in the parent module.

Reply:
We did not call the parent before that, so I don't understand why the parent override it. 🤔 

The link you shared it is indeed the parent method (`default_get`) but it does not override, it uses it. 


I investigated and in fact, the JS code will change the `active_model` and `active_id` when it will call the action received by the server.